plugins/modules/network_settings_intent.py

```diff
@@ -105,23 +121,23 @@
           ipv4_subnet:
             description: IPv4 Subnet address, example 175.175.0.0.
             type: str
-          ipv4TotalHost:
+          ipv4_total_host:
             description: IPv4 total host is required when ipv4_prefix value is false.
             type: int
           ipv6_address_space:
             description: >
               If the value is false only ipv4 input are required, otherwise both
```

Comment:
Determines whether both IPv6 and IPv4 inputs are required.
If set to false, only IPv4 inputs are required.
If set to true, both IPv6 and IPv4 inputs are required.